src/python/primtall/primtall.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Primtall og effektivitet
+level: 3
```

Comment:
merk at vi har endrer litt struktur i mappene nå, da tags legges i data.yml istedenfor i oppgaven. sjekk repoet for eksempler.

Reply:
I repoet syns jeg at jeg så mye dobbelt av dette. Bare for å klargjøre: Det skal *ut* av .md-fila, sant?

Reply:
ja, vi endret nylig. men noe står dobbelt enda pga de gamle sidene trenger den infoen. vil endres ved bruk av et script ved senere annledning. bare sjekk hvordan vi har gjort det i de andre filene når før du sender PR.